_data/governments.yml

```diff
@@ -484,9 +484,6 @@ Singapore:
   - moexmen
   - opengovsg
 
-Slovakia:
-  - slovak-egov
```

Comment:
https://github.com/slovak-egov still exists. Maybe you're hitting an API limit rather than actual removed orgs